hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java

```diff
@@ -4866,12 +4866,48 @@ public void testCheckAndMutateWithTimeRange() throws IOException {
               .thenPut(put);
       assertFalse(ok);
 
+      ok = table.checkAndMutate(ROW, FAMILY).qualifier(QUALIFIER)
```

Comment:
Could you please add a short comment for every test what the purpose is (or move them into separate, small test cases)?